### ISSUE TITLE
documentation？？

### ISSUE THREAD

**Issue by MapleJes** [Taiizor/ReaLTaiizor#164]
Hello, excuse me, is there any documentation, is there any Chinese documentation, UI has done a good job, like。

**Comment by Taiizor** [comment]
Actually, there are no Chinese or English documents. However, each image you see in the project repository is an example application. You can access these examples from the repository as well.

Most of the images are part of the following project, which hosts its application:

https://github.com/Taiizor/ReaLTaiizor/tree/develop/demo/ReaLTaiizor.UI

**Comment by 1124541815** [comment]
It would be nice to have documentation in English or Chinese

**Comment by 1124541815** [comment]
> 事实上，没有中文或英文文件。但是，您在项目存储库中看到的每个图像都是一个示例应用程序。您也可以从存储库访问这些示例。
> 
> 大多数图像都是以下项目的一部分，该项目托管其应用程序：
> 
> https://github.com/Taiizor/ReaLTaiizor/tree/develop/demo/ReaLTaiizor.UI

It would be nice to have documentation in English or Chinese

**Comment by Taiizor** [comment]
> > 事实上，没有中文或英文文件。但是，您在项目存储库中看到的每个图像都是一个示例应用程序。您也可以从存储库访问这些示例。
> > 大多数图像都是以下项目的一部分，该项目托管其应用程序：
> > https://github.com/Taiizor/ReaLTaiizor/tree/develop/demo/ReaLTaiizor.UI
> 
> It would be nice to have documentation in English or Chinese

Yes, having documentation would be nice, but i dont have time for it.
